lib/api/holdings_api.rb

```diff
@@ -30,12 +30,15 @@ class HoldingsAPI < Sinatra::Base
 
     access_count = overlap_records.map(&:access_count).reduce(:+)
     copy_count = overlap_records.map(&:copy_count).reduce(:+)
+    withdrawn_count = overlap_records.map(&:wd_count).reduce(:+)
+    non_withdrawn_count = copy_count - withdrawn_count
```

Comment:
I am checking with @angelinanz whether lost/missing items should be eligible. If so, I think this is the right name and the right calculation. Otherwise, we might want to say "current_holdings_count", and it would need to be calculated as `copy_count - lost_missing_count - withdrawn_count` (yuck)